scripts/gcp_setup/environment/gcloud_env.py

```diff
@@ -683,13 +687,14 @@ def setup_cloudsql_name(self):
         """Ask user to name their Cloud SQL instance."""
         self._print_banner('Setup Cloud SQL name')
         instance_name = raw_input(
-            'Enter a name for the Forseti Cloud SQL instance '
+            'Enter a prefix for the Forseti Cloud SQL instance '
```

Comment:
What prefix would this be?  Offhand, I am not sure myself.  Would it help to provide an example in the displayed output?